### PR TITLE
Bug-fix: Missing strings from Minecraft landing page

### PR DESCRIPTION
**Pull request by nkiruka**
The following strings are missing from the Minecraft landing page due to the strings being in the hourofcode directory:  

- string # 1 
“In partnership with…”  

- string # 2 
“We thank Microsoft for supporting our vision and mission to ensure every child has the opportunity to learn computer science and the skills to succeed in the 21st century.” is missing from Crowdin entirely.

- string # 3
In addition, the string "Minecraft Voyage Aquatic" exists in the code.org directory and it is translated but the translated version is not visible on the Minecraft landing page

### To-do List
- [x] Add strings to i18n gsheet
      - hoc2018_mc_with_microsoft
      - hoc2018_mc_thank_microsoft
- [x] Update parts of code using the strings in the hour of code directory with the new strings added to the code.org directory
- [x] Delete strings from hour of code project
- [x] Remove hoc_s method from minecraft.haml
- [x] Verify strings are visible in Crowdin

### CODE REVIEW

**Comment by joshlory** [comment]
In the future, this sort of PR would be a good candidate for rebase! (Don't worry about changing this one now that you've already pushed it.)

**Comment by joshlory** [comment]
Or could cherry-pick commit c101a7b06326cfa2f49dd859e7feff164d53af35 to a new branch and force-push to `origin/add-missing-mc-strings-to-codeorg-prjt`.

**Review by Hamms — CHANGES_REQUESTED** [review]
Did these strings also get added to the i18n gsheet? Note that gsheet is the source of truth for pegasus i18n strings, so just adding them here is insufficient.

Note that we also need to update the places within code.org pages that are using the hourofcode strings to instead use these ones, and also remove the strings from the hourofcode project if we can

**Comment by nkiruka** [comment]
@Hamms I updated the relevant files with the new strings.  Thanks!

**Comment by nkiruka** [comment]
@Hamms Removed hoc_s method from template.  Thanks!

File: pegasus/sites.v3/code.org/views/minecraft_2018.haml [hunk 1]
```diff
@@ -28,7 +28,7 @@
         .mc-tutorial-info.mc-tutorial-info-wide
           .mc-tutorial-details
             %h2.mc-h1-second-box
-              = hoc_s(:hoc2018_minecraft_aquatic_title1)
+              = I18n.t(:hoc2018_mc_aquatic_title)
               %br/
               = hoc_s(:hoc2018_minecraft_aquatic_title2)
```
Comment (Hamms) [inline pegasus/sites.v3/code.org/views/minecraft_2018.haml:33]: can this be removed?

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 2]
```diff
@@ -27,10 +27,6 @@
 
   hoc2018_tutorial_mchoc_description: 'Minecraft is back for the Hour of Code with a brand new activity! Journey through Minecraft with code.'
 
-  hoc2018_minecraft_aquatic_title1: "Minecraft Voyage Aquatic"
-  hoc2018_minecraft_with_microsoft: "In partnership with"
-  hoc2018_minecraft_thank_microsoft: "We thank Microsoft for supporting our vision and mission to ensure every child has the opportunity to learn computer science and the skills to succeed in the 21st century."
```
Comment (Hamms) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:32]: have you verified that these strings aren't used anywhere else?

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 3]
```diff
@@ -1991,7 +1987,7 @@
...
   tutorial_cmchat_longdescription: "Join a robot monkey in an hour-long course where you will learn the basics of chatbot programing. In the activity, you will advance through a 15 exercise-long course in which you will learn how to program your very own trivia chatbot."
   tutorial_tynkerdb_longdescription: "Choose a dragon, then embark on a quest for treasure. Blast through obstacles and gain cool power-ups as you journey through a mystical jungle. Learn sequencing, loops, conditional logic, and debugging skills along the way."
-  tutorial_cmbuild_longdescription: "Take your coding skills to the next level by learning to build your own games. In this course, you will learn keyboard user-interface and game mechanics as you build a Super Mario™-styled game. Afterwards, you will be able to share your game!" 
+  tutorial_cmbuild_longdescription: "Take your coding skills to the next level by learning to build your own games. In this course, you will learn keyboard user-interface and game mechanics as you build a Super Mario™-styled game. Afterwards, you will be able to share your game!"
```
Comment (Hamms) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:1990]: intentional?

**Review by Hamms — APPROVED** [review]
LGTM, pending some comments!

Note that in order for the existing translations to be applied to the new strings, we need to remove the old strings _after_ the new ones have been added to Crowdin. Fortunately it looks like they went in with today's sync, so we should be good!

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml (hunk 2 above)
Comment (nkiruka) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml:32]: Yes.  These strings are not being used anywhere else.  There is also another string "hoc2018_minecraft_aquatic_title2" which is not being used but it is unrelated to the change in this PR.  A follow-up PR will be opened to delete the string.

File: pegasus/sites.v3/code.org/views/minecraft_2018.haml (hunk 1 above)
Comment (nkiruka) [reply pegasus/sites.v3/code.org/views/minecraft_2018.haml:33]: Good question!  The translation for "Minecraft voyage aquatic" is associated with the key "hoc2018_mc_aquatic_title" in Crowdin.  "hoc2018_minecraft_aquatic_title1" key only existed in the haml file.

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml (hunk 3 above)
Comment (nkiruka) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml:1990]: Yes.  White space was deleted after the removal of three strings that are not being used.

File: pegasus/sites.v3/code.org/views/minecraft_2018.haml (hunk 1 above)
Comment (Hamms) [reply pegasus/sites.v3/code.org/views/minecraft_2018.haml:33]: I'm specifically referring to `hoc2018_minecraft_aquatic_title2` here

Comment (nkiruka) [reply pegasus/sites.v3/code.org/views/minecraft_2018.haml:33]: Yes, it can be removed as well.

**Pull request merged**